.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      TAG_NAME: ${{ github.event.inputs.tag }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: 'temurin'
+          java-version: 17
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          gradle-version: 8.5
+      - name: Build with Gradle
+        id: build
+        run: gradle packageRelease
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        if: startsWith(github.ref, 'refs/tags/')
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I'm not entirely sure if all of them, but by my knowledge actions nowadays have default access to the `GITHUB_TOKEN` so this is no longer required.

Reply:
Thank you for the review ! I wrote this code based on older knowledge. After testing in my repository by removing `token: ${{ secrets.GITHUB_TOKEN }}`, the release worked fine, so I remove this line.